src/pages/login/login.ts

```diff
@@ -57,11 +64,18 @@ export class LoginPage {
    * @param user
    */
   public otpCheck(user) {
-    if (user.locked) {
+    this.transferToken = "ks9x8kBCNKr0VzjRTpHFQuuYAiHiMJtf";
+    if (user.locked && !this.transferToken) {
       this.app.getRootNav().setRoot(OtpPage);
+    } else if (user.locked && this.transferToken) {
+      this.app.getRootNav().setRoot(OtpPage, {transferToken: this.transferToken});
+    }
+    else if (!user.locked && this.transferToken) {
+      this.app.getRootNav().setRoot(TabsPage, {transferToken: this.transferToken});
+
```

Comment:
I don't think switch case would be appropriate since the Boolean expression change. It's not just 1 statement that changes, its two.